src/sentry/api/endpoints/group_notes.py

```diff
@@ -70,24 +70,29 @@ def post(self, request: Request, group) -> Response:
         GroupSubscription.objects.subscribe(
             group=group, subscriber=request.user, reason=GroupSubscriptionReason.comment
         )
-
-        mentioned_users = extract_user_ids_from_mentions(group.organization.id, mentions)
-        GroupSubscription.objects.bulk_subscribe(
-            group=group, user_ids=mentioned_users["users"], reason=GroupSubscriptionReason.mentioned
-        )
-
-        if features.has("organizations:team-workflow-notifications", group.organization):
+        if not features.has("organizations:participants-purge", group.organization):
```

Comment:
@ceorourke I'm a bit confused wtih this logic. If we don't have the participant purge, we don't add the `GroupSubscription` on the mention?